<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MonitoringConfigDlg</class>
 <widget class="QDialog" name="MonitoringConfigDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>监控参数设置</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QListWidget" name="lst_configs">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentRow">
        <number>-1</number>
       </property>
       <item>
        <property name="text">
         <string>监控范围设置</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>温度传感器设置</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>串口设置</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>其他设置</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stack_configs">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page_monitor_scope">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>监控范围设置</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>风炉数量：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spin_boiler_num">
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="maximum">
                 <number>16</number>
                </property>
                <property name="value">
                 <number>7</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>温度传感器数量：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spin_sensor_num">
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="maximum">
                 <number>40</number>
                </property>
                <property name="value">
                 <number>20</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>风机数量：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spin_fan_num">
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="suffix">
                 <string> 个</string>
                </property>
                <property name="maximum">
                 <number>8</number>
                </property>
                <property name="value">
                 <number>4</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_apply_scope">
           <property name="text">
            <string>应用</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="sensor_config">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>温度传感器设置</string>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>风炉：</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" colspan="2">
             <widget class="QComboBox" name="cbo_boiler">
              <property name="currentIndex">
               <number>-1</number>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="3">
             <widget class="Line" name="line">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>最低记录温度：</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QSpinBox" name="spin_low_temp">
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="suffix">
               <string> ℃</string>
              </property>
              <property name="maximum">
               <number>99</number>
              </property>
              <property name="value">
               <number>50</number>
              </property>
             </widget>
            </item>
            <item row="3" column="0" colspan="2">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>最高记录温度：</string>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QSpinBox" name="spin_hi_temp">
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="suffix">
               <string> ℃</string>
              </property>
              <property name="value">
               <number>75</number>
              </property>
             </widget>
            </item>
            <item row="4" column="0" colspan="2">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>报警温度：</string>
              </property>
             </widget>
            </item>
            <item row="4" column="2">
             <widget class="QSpinBox" name="spin_alert_temp">
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="suffix">
               <string> ℃</string>
              </property>
              <property name="value">
               <number>71</number>
              </property>
             </widget>
            </item>
            <item row="5" column="0" colspan="2">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>报警时长：</string>
              </property>
             </widget>
            </item>
            <item row="5" column="2">
             <widget class="QSpinBox" name="spin_alert_time">
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="suffix">
               <string> 秒</string>
              </property>
              <property name="value">
               <number>15</number>
              </property>
             </widget>
            </item>
            <item row="6" column="0" colspan="2">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>巴氏消毒时间：</string>
              </property>
             </widget>
            </item>
            <item row="6" column="2">
             <widget class="QSpinBox" name="spin_pasteu_time">
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="suffix">
               <string> 分钟</string>
              </property>
              <property name="value">
               <number>35</number>
              </property>
             </widget>
            </item>
            <item row="7" column="0" colspan="2">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>消毒完成报警时长：</string>
              </property>
             </widget>
            </item>
            <item row="7" column="2">
             <widget class="QSpinBox" name="spin_pasteu_alert_time">
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="suffix">
               <string> 秒</string>
              </property>
              <property name="value">
               <number>20</number>
              </property>
             </widget>
            </item>
            <item row="8" column="0" colspan="2">
             <widget class="QCheckBox" name="chk_hi_temp_alert">
              <property name="text">
               <string>温度过高自动报警</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <property name="tristate">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_apply_sensor">
           <property name="text">
            <string>应用</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>41</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_com">
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>串口设置</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>选择串口：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="cbo_com">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>波特率：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="cbo_baudrate">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <item>
                 <property name="text">
                  <string>115200</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>19200</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>14400</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>9600</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>采集间隔时间：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spin_acq_interval">
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="suffix">
                 <string> 秒</string>
                </property>
                <property name="value">
                 <number>1</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QLabel" name="label_15">
                <property name="text">
                 <string>串口超时：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spin_timeout">
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="suffix">
                 <string> 毫秒</string>
                </property>
                <property name="maximum">
                 <number>100000</number>
                </property>
                <property name="singleStep">
                 <number>100</number>
                </property>
                <property name="value">
                 <number>1000</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_apply_com">
           <property name="text">
            <string>应用</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>198</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_others">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>其他信息</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>公司信息：</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="edit_company_info"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_apply_others">
           <property name="text">
            <string>应用</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>228</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MonitoringConfigDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MonitoringConfigDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
